### PR TITLE
Create a consistent test environment

### PR DESCRIPTION
**Pull request by benbowler**
Clear automated_testing posts before and after each test feature and with given function

https://trello.com/c/J7nEbLKw/30-3-consider-process-creating-consistent-state-to-run-tests-against

### CODE REVIEW

File: tests/features/sync.feature [hunk 1]
```diff
@@ -15,22 +20,37 @@ Feature: Sync
...
     Given I push all posts from the category sitemap "http://www.shortlist.com/sitemap/food-drink.xml" with the update method as "delete-insert" to the queue
     Then I should have a valid queue ID
-    Then I purge the queue
 
 
-  Scenario: Sync a post from url
+  Scenario: Sync a new post from url
     Given I process the queue action json '{"job":"importUrl","data":{"url":"http:\/\/www.shortlist.com\/food-drink\/michelin-star-restaurants-odd-unusual-world-uk-guide-food","onExistAction":"update"}}'
-    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food" post
+    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food-2" post
```
Comment (ecoad) [inline tests/features/sync.feature]: Why should the `-2` happen on a new post?

File: tests/features/sync.feature [hunk 2]
```diff
@@ -15,22 +20,37 @@ Feature: Sync
...
+  Scenario: Sync a post from url using skip
+    Given I process the queue action json '{"job":"importUrl","data":{"url":"http:\/\/www.shortlist.com\/food-drink\/michelin-star-restaurants-odd-unusual-world-uk-guide-food","onExistAction":"skip"}}'
+    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food-2" post
+    And I should have the identical the post created and updated time
+
+
   Scenario: Get the full import status
     Given I retrieve the full import status
-    Then I should see a couple of imported out of at least 10000
+    Then I should see 100 imported out of at least 10000
```
Comment (ecoad) [inline tests/features/sync.feature]: How is it `100`? I can't see the code that will consistently have the 100 imported posts in the DB?

File: tests/features/sync.feature [hunk 3]
```diff
@@ -15,22 +20,37 @@ Feature: Sync
...
 
   Scenario: Get a category import status
-    Given I retrieve the "news" category import status
-    Then I should see a couple of imported out of at least 5
+    Given I retrieve the "http://www.shortlist.com/sitemap/news.xml" category import status
+    Then I should see 3 imported out of at least 5
```
Comment (ecoad) [inline tests/features/sync.feature]: How is this consistently `3`?

File: tests/features/sync.feature (hunk 1 above)
Comment (benbowler) [inline tests/features/sync.feature]: It should not, this was corrected in another branch. I'll update here too.

File: tests/features/sync.feature (hunk 2 above)
Comment (benbowler) [inline tests/features/sync.feature]: This is an adaptation of the previous I should see *more than* x. As the database changes whenever it is reset from live we can't specify the direct number so I'll update to use the term "more than".

File: tests/features/sync.feature (hunk 3 above)
Comment (benbowler) [inline tests/features/sync.feature]: This is an adaptation of the previous I should see more than x. As the database changes whenever it is reset from live we can't specify the direct number so I'll update to use the term "more than".

**Pull request merged**